src/main.cpp

```diff
@@ -31,8 +33,14 @@ std::string GetEnv(std::string key)
 int main(int argc, char* argv[])
 {
   HABackend backend;
+  g_log.setLogLevel(Logger::LogLevel::Debug);
+  g_log.setDoDetails(true);
+  g_log << Logger::LogLevel::Info << "Starting!" << std::endl;
   if (backend.Connect(GetEnv("HA_WS_URL"), GetEnv("HA_API_TOKEN"))) {
-    cerr << "Connect succesful. Starting." << endl;
+    g_log << Logger::LogLevel::Error << "Connected to HA succesfully!" << std::endl;
+    g_log << Logger::LogLevel::Warning << "Connected to HA succesfully!" << std::endl;
+    g_log << Logger::LogLevel::Info << "Connected to HA succesfully!" << std::endl;
+    g_log << Logger::LogLevel::Debug << "Connected to HA succesfully!" << std::endl;
```

Comment:
```suggestion
    g_log << Logger::LogLevel::Info << "Connected to HA successfully!" << std::endl;
```

(remove 3 levels plus fix a typo)

Reply:
Yes, this was 'testing code'. Will remove.